mon-pix/app/components/module/element/flashcards.gjs

```diff
@@ -1,16 +1,79 @@
+import PixButton from '@1024pix/pix-ui/components/pix-button';
+import { action } from '@ember/object';
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import { t } from 'ember-intl';
+import { eq } from 'ember-truth-helpers';
+import htmlUnsafe from 'mon-pix/helpers/html-unsafe';
 
 export default class ModulixFlashcards extends Component {
-  get currentCard() {
-    return this.args.flashcards.cards[0];
+  @tracked
+  state = 'recto'; // 'intro' | 'recto' | 'verso' | 'outro'
+
+  get currentCardSide() {
+    const side = this.state;
+    return this.args.flashcards.cards[0][side];
+  }
+
+  @action
+  swipeCard() {
+    if (this.state === 'recto') {
+      this.state = 'verso';
+    } else if (this.state === 'verso') {
+      this.state = 'recto';
+    }
   }
 
   <template>
-    <img src={{this.currentCard.recto.image.url}} />
-    {{this.currentCard.recto.text}}
-    <button type="button">{{t "pages.modulix.buttons.flashcards.seeAnswer"}}</button>
-    <p>{{t "pages.modulix.flashcards.direction"}}</p>
-    <p>{{t "pages.modulix.flashcards.position" currentCardPosition=1 totalCards=1}}</p>
+    <div class="element-flashcards">
+      <div class="element-flashcards__card">
+        {{#if this.currentCardSide.image}}
+          <div class="element-flashcards-card__image">
+            <img src={{this.currentCardSide.image.url}} alt="" />
+          </div>
+        {{/if}}
+
+        <div class="element-flashcards-card__text">
+          {{#if (eq this.state "recto")}}
+            <p class="element-flashcards-card__text--recto">{{this.currentCardSide.text}}</p>
+          {{else if (eq this.state "verso")}}
+            {{htmlUnsafe this.currentCardSide.text}}
+          {{/if}}
+        </div>
+
+        <div class="element-flashcards-card__footer element-flashcards-card__footer--{{this.state}}">
+          {{#if (eq this.state "recto")}}
+            <PixButton
+              class=""
+              @triggerAction={{this.swipeCard}}
+              @variant="primary"
+              @size="small"
+              @iconAfter="rotate-right"
+            >
+              {{t "pages.modulix.buttons.flashcards.seeAnswer"}}
+            </PixButton>
+          {{/if}}
+          {{#if (eq this.state "verso")}}
+            <PixButton class="" @triggerAction={{this.swipeCard}} @variant="tertiary" @size="small">
+              {{t "pages.modulix.buttons.flashcards.seeAgain"}}
+            </PixButton>
+          {{/if}}
+        </div>
+      </div>
+
+      <div class="element-flashcards__footer">
+        {{#if (eq this.state "recto")}}
+          <p class="element-flashcards-footer__direction">{{t "pages.modulix.flashcards.direction"}}</p>
+          <p class="element-flashcards-footer__position">{{t
+              "pages.modulix.flashcards.position"
+              currentCardPosition=1
```

Comment:
**suggestion**: Si on ajoute un ajoute un prop `currentCardIndex` comme proposé plus haut, on peut aussi ajouter un getter `get currentCardPosition` qui renvoie `this.currentCardIndex + 1`.

Reply:
Je propose de laisser tel quel pour l'instant, et on pourra voir cela pour le ticket de _afficher plusieurs cartes_